swift_browser_ui_frontend/src/components/DeleteModal.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <c-card class="delete-modal">
+    <c-alert type="error">
+      <div slot="title">
+        {{ title }}
+      </div>
+
+      {{ message }}
+
+      <c-card-actions justify="end">
+        <c-button 
+          outlined
+          @click="toggleDeleteModal"
+          @keyup.enter="toggleDeleteModal"
+        >
+          Cancel
+        </c-button>
```

Comment:
We have `Cancel` term translated in lang.js. Could we also add it here?

Reply:
ok